UtahModules/Sources/UtahTrends/DataCard.swift

```diff
@@ -54,11 +58,28 @@ struct DataCard: View {
                 .shadow(radius: 5)
         }
         .task {
-            await firestoreManager.loadObservations(metricCode: "55423-8")
-            recalculateChartData(basedon: firestoreManager.observations)
+            if title == "Daily Step Count" {
+                await firestoreManager.loadObservations(metricCode: "55423-8")
+                recalculateChartData(basedon: firestoreManager.observations)
+            } else if title == "Edmonton Frail Scale" {
+                await firestoreManager.loadSurveys()
+                getSurveyData(surveyType: "edmonton")
+            } else if title == "Veines Survey Score" {
```

Comment:
this should also have a case for 'wiq' right?

Reply:
we have 3 surveys

Reply:
I just changed this in a separate PR!

Reply:
^ update: this change is in this PR